lxd/containers_post.go

```diff
@@ -42,6 +42,16 @@ func instanceTypeToDBType(arg string) (instance.Type, error) {
 	return -1, fmt.Errorf("Invalid instance type")
 }
 
+// instanceDBTypeToString converts a native DB representation of instance type to a string used
+// in API requests. Returns empty string if instance type supplied is invalid.
+func instanceDBTypeToString(arg instance.Type) string {
```

Comment:
Feels like the wrong spot for this, shouldn't the functions to convert from string to dbtype and dbtype to string be in the db package?

Reply:
@stgraber agreed, I've moved the conversion functions into the instance package as it feels more idiomatic there.